src/main/resources/hudson/plugins/jira/JiraIssueUpdateBuilder/config.jelly

```diff
@@ -14,6 +14,9 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+  <f:entry title="${%Update jira issues' status found in changelogs}" field="updateRelatedIssues">
+    <f:checkbox/>
```

Comment:
Looking at the options and on the logic, it is not clear when and when not jqlSearch is used. Apart from the fact that we should avoid nested if's, with this logic I think it would be better to use radio button instead of checkbox here as the jqlSearch will not be used in this case?